### PR TITLE
Feat: Add generateShortUrl() function with .NET 6

### PR DESCRIPTION
**Pull request by adityaoberai**
## Description

Generates a short URL using the `bitly` and `tinyurl` APIs.

### Test 1

- Input 

image

- Output 

image

### Test 2

- Input 

image

- Output 

image

### Test 3

- Input 

image

- Output 

image

### Test 4

- Input 

image

- Output 

image

### Test 5

- Input 

image

- Output 

image

## Issues solved

Solves issue [appwrite/appwrite #4193](https://github.com/appwrite/appwrite/issues/4193)

### CODE REVIEW

File: dotnet/generateShortUrl/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+_Example successful response:_
+
+```json
+{
+    "success":true,
+    "url":"https:\/\/bit.ly\/3DGAJF8"
+}
+```
+
+_Example successful response:_
```
Comment (Meldiron) [inline dotnet/generateShortUrl/README.md:27]:
```suggestion
_Example failure response:_
```

File: dotnet/generateShortUrl/src/Index.cs [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    string apiKey = "";
+    string uri = "";
+    Dictionary<string, string> body = new Dictionary<string, string>();
+    string json = "";
+
+    // Preparing API info based on chosen provider
+
+    if(payload["provider"].Equals("bitly"))
+    {
+      apiKey = req.Variables["BITLY_API_KEY"];
```
Comment (Meldiron) [inline dotnet/generateShortUrl/src/Index.cs:50]: Let's add check that ensures API key is provided.

Comment (Meldiron) [reply dotnet/generateShortUrl/src/Index.cs:50]: Same for tiny URL

**Review by Meldiron — CHANGES_REQUESTED** [review]
Looks great, tiny comments 😇

File: dotnet/generateShortUrl/src/Index.cs (hunk 2 above)
Comment (adityaoberai) [reply dotnet/generateShortUrl/src/Index.cs:58]: Made the changes

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**